src/main/webapp/js/navigation-loader.js

```diff
@@ -31,26 +31,37 @@ function addLoginOrLogoutLinkToNavigation() {
         })
         .then((loginStatus) => {
             if (loginStatus.isLoggedIn) {
-                navigationElement.appendChild(createListItemUl(createListItem(createLink(
-                    '/user-page.html?user=' + loginStatus.username, 'Your Page'))));
+                /**Creation of the menu and each element of the navigation bar **/
+                navigationElement.appendChild(createListItemUl(createLink(
+                    '/user-page.html?user=' + loginStatus.username, createListItem('Profile'))));
```

Comment:
Note: if we change the paths in the backend, these need to be changed too. Can you make a comment in each of the views that navigation-loader.js is a dependency?

Reply:
Sorry ryan I forgot to change this

Reply:
did you make a comment in the views?